packages/plugin-flex/src/commands/flex/plugins/archive/plugin.ts

```diff
@@ -24,10 +25,20 @@ export default class FlexPluginsArchivePlugin extends ArchiveResource<Plugin> {
     }),
   };
 
-  protected _parsedFlags?: OutputFlags<typeof FlexPluginsArchivePlugin.flags>;
+  public _flags: OutputFlags<typeof FlexPluginsArchivePlugin.flags>;
+
+  constructor(argv: string[], config: ConfigData, secureStorage: SecureStorage) {
+    super(argv, config, secureStorage);
+    this._flags = {
```

Comment:
Why are you setting all of these and then overriding them? Is this a TypeScript thing? If so, please just add `// @ts-ignore` or fix the issue. This is not a good pattern at all.

Reply:
Fixed in [5fda251](https://github.com/twilio/flex-plugin-builder/pull/722/commits/5fda25124004c3149a9536ca38e88f225655bee2)